src/core/utils.js

```diff
@@ -494,6 +495,25 @@ export function flatten(collector, item) {
 
 // --- DOM HELPERS -------------------------------
 
+/**
+ * Separates each item with proper commas and "and".
+ * @param {string[]} array
+ * @param {(str: string) => object} mapper
+ */
+export function htmlJoinAnd(array, mapper = item => item) {
+  const items = array.map(mapper);
+  switch (items.length) {
+    case 0:
+    case 1: // "x"
+      return hyperHTML`${items[0]}`;
```

Comment:
```suggestion
      return items[0];
```

Another final nit (will commit this myself)